### PR TITLE
Select html

### PR DESCRIPTION
**Pull request by tisidro**
#Description
This is a documentation of learning as I work through a [Zastrow CSS tutorial](https://www.digitalocean.com/community/tutorials/how-to-work-with-the-box-model-in-css)
##Validation
- [ ] Code is proofread.
- [ ] Documents are formatted.
- [ ] Commit messages are appropriate.
Thank you for helping with my progress in Github workflow and CSS mastery! :)

### CODE REVIEW

File: 2select-html/styles.css [hunk 1]
```diff
@@ -2,15 +2,47 @@ body {
     font-family: sans-serif;
```
Comment (arnest00) [inline 2select-html/styles.css]: Extra spaces here, for example

File: 2select-html/styles.css [hunk 2]
```diff
@@ -2,15 +2,47 @@ body {
...
     font-weight: normal;
 }
 
-h3 {
+h2 {
+    color: maroon;
+}
+
+/* h3 {
     font-weight: normal;
```
Comment (arnest00) [inline 2select-html/styles.css]: You can delete this commented code, as the styling is implemented in the block starting on line 5

File: 2select-html/styles.css [hunk 3]
```diff
@@ -2,15 +2,47 @@ body {
...
+
+strong {
+  color: red; 
+}
+
+p strong {
+
+    color: coral;
+
+}
```
Comment (arnest00) [inline 2select-html/styles.css:25]: Extra line breaks

File: 2select-html/styles.css [hunk 4]
```diff
@@ -2,15 +2,47 @@ body {
...
+/* h3 {
     font-weight: normal;
-} */
+}  */
 
 h2,
 h3 {
      font-weight: normal;
+}
+
```
Comment (arnest00) [inline 2select-html/styles.css:14]: You can delete lines 19-22 for the same reason.

**Review by arnest00 — CHANGES_REQUESTED** [review]
Great job, Terri - looks like all the CSS has been used correctly. A few things to note:

- There is some inconsistent indentation in the CSS file. You could check your code editor settings and see what your tab size is set to. I commented one example in `styles.css`.
- There is also some duplicated code I don't think you were intending on keeping in there.

File: 2select-html/styles.css (hunk 1 above)
Comment (tisidro) [reply 2select-html/styles.css]: Thanks, I am checking this out and will fix. I appreciate the suggestion to check my tab size in the editor!

File: 2select-html/styles.css (hunk 3 above)
Comment (tisidro) [reply 2select-html/styles.css:25]: fixed! :)

File: 2select-html/styles.css (hunk 2 above)
Comment (tisidro) [reply 2select-html/styles.css]: done! :)

**Review by arnest00 — APPROVED** [review]

**Pull request merged**